projects/cm_mcu/InitTask.c

```diff
@@ -39,12 +40,15 @@ void InitTask(void *parameters)
   while (getPowerControlState() != POWER_ON) {
     vTaskDelay(pdMS_TO_TICKS(1000));
   }
-#endif                  // REV1
-  init_registers_ff();  // initialize I/O expander for fireflies -- with FF monitoring via I2C in other threads, it grabs semaphore inside
-  init_registers_clk(); // initialize I/O expander for clocks
+#endif                               // REV1
+  init_registers_ff();               // initialize I/O expander for fireflies -- with FF monitoring via I2C in other threads, it grabs semaphore inside
+  int status = init_registers_clk(); // initialize I/O expander for clocks
   readFFpresent();
-  log_info(LOG_SERVICE, "Clock I/O expander initialized\r\n");
-
+  if (status == 0)
+    log_info(LOG_SERVICE, "Clock I/O expander initialized\r\n");
+  else {
+    log_info(LOG_SERVICE, "Clock I/O expander failed\r\n");
+  }
```

Comment:
should we add something to the EEPROM error logger too?

Reply:
Done. (I have also fixed some bugs from the error logging voltage alarm). There are a few minor things I would like to change from this task and I will open a new branch for this.